tests/columns/test_bool.py

```diff
@@ -0,0 +1,73 @@
+from tests.testcase import BaseTestCase
+from tests.util import require_server_version
+from clickhouse_driver import errors
+
+
+class BoolTestCase(BaseTestCase):
+    @require_server_version(21, 12)
```

Comment:
Since every test requires similar server version, it's better to use class attribute: `required_server_version = (X, Y)`. There is no need to use decorator `require_server_version` on every test.